gammapy/stats/variability.py

```diff
@@ -228,3 +230,102 @@ def compute_flux_doubling(flux, flux_err, coords, axis=0):
     }
 
     return doubling_dict
+
+
+def TKAlgorithm(
```

Comment:
I would use an explicit name. The name should convey the information on what the function does.